### ISSUE TITLE
Plugin initialization error

### ISSUE THREAD

**Issue by Daztek** [PhilippeChab/nwscript-ee-language-server#66]
Hey, my VSCode recently updated to the following version and now I'm getting the following plugin error.

```
Version: 1.86.0 (user setup)
Commit: 05047486b6df5eb8d44b2ecd70ea3bdf775fd937
Date: 2024-01-31T10:28:19.990Z
Electron: 27.2.3
ElectronBuildId: 26495564
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Windows_NT x64 10.0.22631
```

![image](https://github.com/PhilippeChab/nwscript-ee-language-server/assets/31415508/c721a1b3-175e-459d-b656-a671d1445fe3)

**Comment by ReachPW** [comment]
I'm getting the same after update

```Version: 1.86.0
Commit: 05047486b6df5eb8d44b2ecd70ea3bdf775fd937
Date: 2024-01-31T10:27:46.147Z
Electron: 27.2.3
ElectronBuildId: 26495564
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Linux x64 6.1.0-17-amd64```

.


`Request initialize failed with message: Near offset 67732: unexpected opened tag /dict ~~~ </array> </dict> ~~~`

**Comment by tinygiant98** [comment]
Getting the same.  Seems to be some kind of issue with parsing the textmate grammar.  The offests appear to be random, but are always associated with `</dict>` tag.  My offset errors (this time) are at 78303, 82922, 85057, 80564 and 88529.   Seem to be different every time, so I don't think there's an issue with the grammar.  Must be something else ...

**Comment by PhilippeChab** [comment]
Thanks for the report.

I'll look into this when I have some free time.

**Comment by PhilippeChab** [comment]
Fixed in 2.1.1.
